File: P1/E1/BicicletaCarretera.java
package E1;

public class BicicletaCarretera extends Bicicleta {

    BicicletaCarretera(int id){
        super(id);
    }

    @Override
    public void pedalear(){
        System.out.println("Corre una bicicleta de carretera de id: " + super.id);
    }

}

File: P1/E2/BicicletaCarretera.py
from Bicicleta import Bicicleta

class BicicletaCarretera(Bicicleta):
    def __init__(self, id):
        super().__init__(id)

    def pedalear(self):
        print("Corre una bicicleta de carretera de id:", self.id)

    def clonar(self):
        return BicicletaCarretera(self.id) #return deepcopy(self)
